canton/community/bindings-java/src/main/java/com/daml/ledger/javaapi/data/CommandsSubmissionV2.java

```diff
@@ -399,6 +399,7 @@ public CommandsOuterClass.Commands toProto() {
             .setApplicationId(applicationId)
             .setCommandId(commandId)
             .addAllCommands(commandsConverted)
+            .setParty(actAs.get(0))
```

Comment:
Why is it always the first?

Reply:
It has been this way before in v1. It doesn't really matter tbh, server makes a superset of party and actAs anyway